includes/admin/payments/refunds.php

```diff
@@ -192,21 +202,33 @@ function edd_refund_details_attributes( $refund ) {
 	</h2>
 
 	<div class="edd-admin-box-inside edd-admin-box-inside--row">
-		<span>
-			<?php esc_html_e( 'Created', 'easy-digital-downloads' ); ?>
-		</span>
-		<time datetime="<?php echo esc_attr( EDD()->utils->date( $refund->date_created, null, true )->toDateTimeString() ); ?>">
-			<?php echo edd_date_i18n( $refund->date_created, 'M. d, Y' ) . '<br />' . edd_date_i18n( strtotime( $refund->date_created ), 'H:i' ); ?> <?php echo esc_html( edd_get_timezone_abbr() ); ?>
-		</time>
+		<div class="edd-form-group">
+			<span class="edd-form-group__label">
+				<?php esc_html_e( 'Original Order', 'easy-digital-downloads' ); ?>
+			</span>
+			<div class="edd-form-group__control">
+				<a href="<?php echo esc_url( $order_url ); ?>"><?php echo esc_html( $order->number ); ?></a>
+			</div>
+		</div>
 	</div>
 
-	<div class="edd-admin-box-inside edd-admin-box-inside--row">
-		<span>
-			<?php esc_html_e( 'Original Order', 'easy-digital-downloads' ); ?>
-		</span>
-		<span>
-			<a href="<?php echo esc_url( $order_url ); ?>"><?php echo esc_html( $order->number ); ?></a>
-		</span>
+	<div class="edd-admin-box-inside">
+		<div class="edd-form-group">
+			<span class="edd-form-group__label">
+				<?php esc_html_e( 'Created', 'easy-digital-downloads' ); ?>
+			</span>
+			<div class="edd-form-group__control">
+				<time datetime="<?php echo esc_attr( EDD()->utils->date( $refund->date_created, null, true )->toDateTimeString() ); ?>" style="line-height: normal">
```

Comment:
I've noticed this in other places where EDD is displaying the date and/or time, but is it possible for the date/time to use the format set in General > Settings? I've changed mine a couple different ways to experiment but this appears to be hard coded instead of using my preferences.

Reply:
@robincornett I took the same formatting that is used on the main "Orders" table for the "Date" column. There is some discussion around timezone and dates here: https://github.com/easydigitaldownloads/easy-digital-downloads/issues/7016

The actual format could be changed to your site preference, but for the time being I'm using the Orders table as the source of truth for how to output the date and time.

Reply:
Yes, I asked at one point about creating an issue with the tables not honoring the user settings for dates and times...I think we should honor the settings, with regional differences in how dates are displayed, and 24 hour times potentially confusing some (at least me).